### PR TITLE
[upmeter] Get rid of shell-operator dependency

### PR DESCRIPTION
**Pull request by nabokihms**
Signed-off-by: m.nabokikh <maksim.nabokikh@flant.com>

## Description
* Remove shell-operator dependency
* Rewrite the code to use plain client-go shared informers

## Why do we need it, and what problem does it solve?
Shell-operator's `monitor` abstraction is not required for upmeter use cases. It only adds obscurity because of the cumbersome interface.  This slight improvement is aimed at making the debugging process simpler and reducing dependency management's complexity.

## Checklist
- [x] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: upmeter
type: fix
summary: Get rid of shell-operator dependency
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @nabokihms. I've started the workflow for label 'e2e/run/gcp'...

File: modules/500-upmeter/images/upmeter/pkg/monitor/node/monitor_test.go [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+		t.Fatalf("failed to start the nodes monitor: %v", err)
+	}
+
+	// #1 Fresh start
+	nodes, err := monitor.List()
+	if err != nil {
+		t.Fatalf("failed to get nodes: %v", err)
+	}
+	if len(nodes) != 1 {
+		t.Fatalf("there should be only one nodes on the start: actual quantity %d", len(nodes))
```
Comment (shvgn) [inline modules/500-upmeter/images/upmeter/pkg/monitor/node/monitor_test.go:70]:
```suggestion
		t.Fatalf("there should be a single node on the start, got %d", len(nodes))
```

File: modules/500-upmeter/images/upmeter/pkg/monitor/node/monitor_test.go [hunk 2]
```diff
@@ -0,0 +1,112 @@
+/*
+Copyright 2021 Flant JSC
```
Comment (shvgn) [inline modules/500-upmeter/images/upmeter/pkg/monitor/node/monitor_test.go:2]:
```suggestion
Copyright 2022 Flant JSC
```

**Review by shvgn — APPROVED** [review]
Did some chore, LGTM overall

**Pull request merged**